### PR TITLE
Better formatting

### PR DESCRIPTION
**Pull request by plally**
Indentation, parentheses, spacing and language items for the cleanup commands

### CODE REVIEW

File: lua/entities/marums_hoverball/init.lua [hunk 1]
```diff
@@ -73,14 +75,15 @@ function ENT:PhysicsUpdate()
...
-    endpos = self:GetPos()+Vector(0,0,-hoverdistance*2),
-    filter = function( ent ) if ( ent:GetClass() == "prop_physics" ) then return false end end,
-    mask = detectmask
-    } )
 
-    local distance = self:GetPos():Distance(tr.HitPos)
+    local tr = util.TraceLine{
+        start = self:GetPos(),
+        endpos = self:GetPos() + Vector(0, 0, -hoverdistance * 2 ),
+        filter = function( ent ) if ( ent:GetClass() == "prop_physics" ) then return false end end,
```
Comment (sarahsturgeon) [inline lua/entities/marums_hoverball/init.lua]: Man I don't like how this section is formatted at all. I think `endpos` and `filter` should be defined above `tr` in a better way. Endpos could be broken into two steps to define what the Vector is. Filter should be split up into multiple lines.

File: lua/weapons/gmod_tool/stools/hoverball_spawner.lua [hunk 2]
```diff
@@ -26,10 +27,12 @@ function TOOL:LeftClick( trace )
         entity.detectswater =       self:GetClientNumber( "detects_water" )
         entity:SetModel(            self:GetClientInfo(   "model" ) )
         entity:Spawn()
-        if (IsValid(trace.Entity)) then
+        if IsValid(trace.Entity) then
```
Comment (sarahsturgeon) [inline lua/weapons/gmod_tool/stools/hoverball_spawner.lua]:
```suggestion
        if IsValid( trace.Entity ) then
```

**Review by sarahsturgeon — APPROVED** [review]
Approving so you can rebase and merge after addressing my comments 👍

File: lua/entities/marums_hoverball/init.lua (hunk 1 above)
Comment (plally) [reply lua/entities/marums_hoverball/init.lua]: I agree. im not even sure the filter does anything.

Comment (sarahsturgeon) [reply lua/entities/marums_hoverball/init.lua]: Lol, of course it doesn't... 🤦‍♂

**Pull request merged**